### PR TITLE
Translate phrases to emoji command

### PR DESCRIPTION
**Pull request by MoPaMo**
Based on my [issue#1312](https://github.com/SwitchbladeBot/switchblade/issues/1312), I have now created a command, which takes user input and translates the words into emojis. Right now, the command has a pretty small range of translations, but I think finding new and fun translation is a great thing for PRs.

I've just tested the algorithm itself and haven't run the entire project myself yet, so I'm not sure whether I "registered" my command...

### CODE REVIEW

File: src/commands/text/emojitr.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  twitter: ":bird:",
+  up: ":arrow_up: ",
+  watch: ":watch:",
+  well: ":whale2:",
+  whale: ":whale2:",
+  yes: ":ballot_box_with_check: ",
+  you: ":point_right: :bust_in_silhouette:",
+  youtube: ":arrow_forward: :pause_button:",
+};
+const seperators = /([\.,\!\?;])/;
```
Comment (almeidx) [inline src/commands/text/emojitr.js:69]:
```suggestion
const separators = /([\.,\!\?;])/;
```

File: src/commands/text/emojitr.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      client
+    );
+  }
+
+  async run({ t, author, channel }, text) {
+    const emojified = text
+      .split(" ")
+      .map((word) => {
+        let suffix = seperators.test(word) ? seperators.exec(word)[1] : "";
+        word = word.replace(seperators, "");
```
Comment (almeidx) [inline src/commands/text/emojitr.js:94]:
```suggestion
        let suffix = separators.test(word) ? separators.exec(word)[1] : "";
        word = word.replace(separators, "");
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — COMMENTED** [review]
I feel like it would be better if you used the emojis directly (😅) instead of keeping the `:emoji_name:` format

File: src/commands/text/emojitr.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  twitter: ":bird:",
+  up: ":arrow_up: ",
+  watch: ":watch:",
+  well: ":whale2:",
+  whale: ":whale2:",
+  yes: ":ballot_box_with_check: ",
+  you: ":point_right: :bust_in_silhouette:",
+  youtube: ":arrow_forward: :pause_button:",
+};
+const separators = /([\.,\!\?;])/;
```
Comment (almeidx) [inline src/commands/text/emojitr.js:69]: No need to escape these characters inside a character set
```suggestion
const separators = /([.,!?;])/;
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Comment by MoPaMo** [comment]
> I feel like it would be better if you used the emojis directly (😅) instead of keeping the `:emoji_name:` format

Why? They are converted to emojis by discord and I'm not even sure if the "normal" emojis work…

**Comment by almeidx** [comment]
> > I feel like it would be better if you used the emojis directly (sweat_smile) instead of keeping the `:emoji_name:` format
> 
> Why? They are converted to emojis by discord and I'm not even sure if the "normal" emojis work…

It would probably make the code easier to read, but it's not a big deal

**Comment by MoPaMo** [comment]
I  don't know 🤷🏼‍♂️ 😂

**Comment by almeidx** [comment]
[Github Actions are sort of dying atm](https://www.githubstatus.com/incidents/zbpwygxwb3gw) so the checks aren't running, however, whenever they get back up they are going to fail in this pull request because we do not allow semicolons. 

For the time being, you can run `npm run fix` and `npm run test` locally, which is basically what the github actions would have done if they were working

**Comment by MoPaMo** [comment]
I fixed the problems[ called out by the linter program](https://github.com/SwitchbladeBot/switchblade/runs/2595080291?check_suite_focus=true), so what do I do now?

**Comment by MoPaMo** [comment]
I‘m kind of frustrated right now - what did I do wrong so I don’t get any answer? 🙈

**Comment by Doges** [comment]
> 
> 
> I‘m kind of frustrated right now - what did I do wrong so I don’t get any answer in two weeks? 🙈

nothing, i don't have enough time to test it rn, may take more 4 days for my review

**Comment by MoPaMo** [comment]
> > I‘m kind of frustrated right now - what did I do wrong so I don’t get any answer in two weeks? 🙈
> 
> nothing, i don't have enough time to test it rn, may take more 4 days for my review

Sorry for being annoying 😅

File: src/commands/text/emojitr.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+      },
+      client
+    )
+  }
+
+  async run ({ t, author, channel }, text) {
+    const emojified = text
+      .split(' ')
+      .map((word) => {
+        const suffix = separators.test(word) ? separators.exec(word)[1] : ''
```
Comment (almeidx) [inline src/commands/text/emojitr.js:93]:
```suggestion
        const suffix = separators.exec(word)?.[1] ?? ''
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Pull request closed without merging**